src/main/java/picard/fingerprint/FingerprintChecker.java

```diff
@@ -402,7 +438,7 @@ public IntervalList getLociToGenotype(final Collection<Fingerprint> fingerprints
         final SamReader in = SamReaderFactory.makeDefault()
                 .enable(SamReaderFactory.Option.CACHE_FILE_BASED_INDEXES)
                 .referenceSequence(referenceFasta)
-                .open(samFile);
+                .open(samFile, null, seekableChannelFunction);
```

Comment:
I would add a comment here describing why this is needed and pointing at the htsjdk issue number ( is there one... there should be.)